packages/viewer/src/components/viewer-dom-renderer/viewer-dom-renderer.tsx

```diff
@@ -39,6 +41,16 @@ export class ViewerDomRenderer {
   @Prop()
   public drawMode: ViewerDomRendererDrawMode = '3d';
 
+  /**
+   * Specifies whether or not to propogate events to the viewers' interaction handlers
```

Comment:
Nit: Could change to `Specifies whether to propagate events to the viewer's interaction handlers`